### ISSUE TITLE
Unicode warnings when running the brakeman and rubocop rake tasks together

### ISSUE THREAD

**Issue by dylnclrk** [rubocop-hq/rubocop#7475]
⚠️ *Note: I filed this as an issue on the Brakeman project (https://github.com/presidentbeef/brakeman/issues/1424)*

*When running the rubocop and brakeman rake tasks together, I see a bunch of Unicode warnings in the console.* 

--------

## Expected behavior

Adding Rubocop and Brakeman tasks to my project's default Rake task should not print warnings.

## Actual behavior

Adding Rubocop and Brakeman tasks to my project's default Rake task outputs a set of unicode warnings. Running them in the other order (Brakeman first, Rubocop second) does not show any warnings.

### The warnings:

```
/…/lib/ruby/gems/2.6.0/gems/brakeman-4.7.1/bundle/ruby/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width/constants.rb:3: warning: already initialized constant Unicode::DisplayWidth::VERSION
/…/lib/ruby/gems/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width/constants.rb:3: warning: previous definition of VERSION was here
/…/lib/ruby/gems/2.6.0/gems/brakeman-4.7.1/bundle/ruby/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width/constants.rb:4: warning: already initialized constant Unicode::DisplayWidth::UNICODE_VERSION
/…/lib/ruby/gems/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width/constants.rb:4: warning: previous definition of UNICODE_VERSION was here
/…/lib/ruby/gems/2.6.0/gems/brakeman-4.7.1/bundle/ruby/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width/constants.rb:5: warning: already initialized constant Unicode::DisplayWidth::DATA_DIRECTORY
/…/lib/ruby/gems/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width/constants.rb:5: warning: previous definition of DATA_DIRECTORY was here
/…/lib/ruby/gems/2.6.0/gems/brakeman-4.7.1/bundle/ruby/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width/constants.rb:6: warning: already initialized constant Unicode::DisplayWidth::INDEX_FILENAME
/…/lib/ruby/gems/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width/constants.rb:6: warning: previous definition of INDEX_FILENAME was here
/…/lib/ruby/gems/2.6.0/gems/brakeman-4.7.1/bundle/ruby/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width/index.rb:9: warning: already initialized constant Unicode::DisplayWidth::INDEX
/…/lib/ruby/gems/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width/index.rb:9: warning: previous definition of INDEX was here
/…/lib/ruby/gems/2.6.0/gems/brakeman-4.7.1/bundle/ruby/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width.rb:6: warning: already initialized constant Unicode::DisplayWidth::DEPTHS
/…/lib/ruby/gems/2.6.0/gems/unicode-display_width-1.6.0/lib/unicode/display_width.rb:6: warning: previous definition of DEPTHS was here
```

## Steps to reproduce the problem

Clone my [example project (dylnclrk/rubocop-brakeman-warning-reproduction)](https://github.com/dylnclrk/rubocop-brakeman-warning-reproduciton), run `bundle install` and `bin/rake` to output the warnings.

*Alternatively*, you could create your own version of the issue:
* `rails new test-reproduction-app`
* add rubocop and brakeman to your Gemfile
* add [`rubocop.rake`](https://github.com/dylnclrk/rubocop-brakeman-warning-reproduciton/blob/af15455b30a32827c9b72d7341a68fe572ed73d2/lib/tasks/rubocop.rake) and [`brakeman.rake`](https://github.com/dylnclrk/rubocop-brakeman-warning-reproduciton/blob/af15455b30a32827c9b72d7341a68fe572ed73d2/lib/tasks/brakeman.rake) tasks.
* modify [`Rakefile`](https://github.com/dylnclrk/rubocop-brakeman-warning-reproduciton/blob/af15455b30a32827c9b72d7341a68fe572ed73d2/Rakefile) to make `task default: %i[rubocop brakeman]`
* run `bin/rake`

## RuboCop version

```
0.76.0 (using Parser 2.6.5.0, running on ruby 2.6.2 x86_64-darwin18)
```

## Other versions

```
Brakeman version: 4.7.1
Rails version: 6.0.0
Ruby version: 2.6.2
```

**Comment by dylnclrk** [comment]
I just took a look at the suggested way to add brakeman and rubocop rake tasks, from the docs, and realized I was not following conventions. Here are the [brakeman docs](https://brakemanscanner.org/docs/rake/), and [rubocop docs](https://docs.rubocop.org/en/stable/integration_with_other_tools/#rake-integration) on setting up rake tasks. 

I updated the sample project's Rakefile to follow the suggestions (https://github.com/dylnclrk/rubocop-brakeman-warning-reproduciton/commit/b3013e0e85c28a397e85dc0f75c17add27413f6e). And I'm still seeing the warnings.

**Comment by dylnclrk** [comment]
Closing this, as it's a byproduct of using what's supposed to be a standalone application in a rake task. 

From @presidentbeef on https://github.com/presidentbeef/brakeman/issues/1424:

> This is because Brakeman vendors its dependencies and then Rubocop is probably also loading the same libraries. If Brakeman goes first, no problem. But if Rubocop goes first, then Brakeman will load its own version of unicode-display_width and the namespaces clash.

@presidentbeef's suggested fix is to use `brakeman-lib` instead, which does not vendor dependencies.
